packages/frontend/src/features/sqlRunner/store/barChartSlice.ts

```diff
@@ -175,6 +163,32 @@ export const barChartConfigSlice = createSlice({
                 config.display.yAxis[index].position = position;
             }
         },
+        addSeries: (
+            state,
+            action: PayloadAction<
+                SqlTransformBarChartConfig['y'][number]['reference']
+            >,
+        ) => {
+            if (!state.config) return;
+            if (!state.config.fieldConfig) return;
+
+            const yAxisFields = state.config.fieldConfig.y;
+            if (yAxisFields) {
+                state.config.fieldConfig.y.push({
+                    reference: action.payload,
+                    aggregation: DEFAULT_AGGREGATION,
+                });
+                state.config.display = state.config.display || {};
+                state.config.display.yAxis = state.config.display.yAxis || [];
+                state.config.display.series = {
+                    ...state.config.display.series,
+                    [action.payload]: {
+                        yAxisIndex: yAxisFields.length,
+                        label: action.payload,
+                    },
+                };
+            }
+        },
```

Comment:
is this what is expected?
When I add a series I also push this new item to the `yAxis`? 🤔